src/pages/ecosystem.md

```diff
@@ -0,0 +1,59 @@
+---
+title: OpenRefine Ecosystem
+description: OpenRefine Ecosystem
+hide_table_of_contents: true
+---
+# OpenRefine Ecosystem
+
+The OpenRefine ecosystem comprises the OpenRefine project, which maintains the OpenRefine software, user forums, documentation, and user groups that contextualize OpenRefine for their communities' usage.
+
+Those user groups include data journalists, semantic web enthusiasts, Galleries, Libraries, Archives, Museums (GLAM), researchers, or members of the Wikimedia Movement. Each community provides advocacy, training, and support activities via mediums relevant to its members, with little coordination between them. For instance, libraries host training sessions for archivists, librarians, and researchers; workshops are organized during data journalism conferences; and a dedicated Telegram channel is available for Wikimedians seeking support.
+
+Third-party developers create new extensions and reconciliation services compatible with OpenRefine. Reconciliation services allow users to link their datasets with a knowledge graph, while extensions add domain-specific features or integrate OpenRefine into a particular workflow. These services expand OpenRefine's capabilities, enabling users to accomplish their tasks more efficiently.
+
+Finally, the Advisory Committee is responsible for OpenRefine funding and administrative tasks to support the ecosystem.
+
+The following graphic represents the relationship between OpenRefine and its user communities. Click on the 🔗 for more details on each node. The blue section represents activities coordinated by the OpenRefine project.
```

Comment:
This seems very (needlessly?) complex for such a small project (one full time developer and a small handful of part timers).

Reply:
What's complex? The sentence or the following mind-map like graphic. My intent here is to capture how things currently operate.

Reply:
In this diagram I struggle a bit to understand what is a node and what is an edge.
The type of concepts that go into nodes doesn't seem to be uniform to me. For instance there are nodes about:
* "user support" (an activity)
* "support forum" (an online space)
* "advisory committee" (a group of people)

Are those meant to always refer to a specific group of people? Maybe the phrasing of the nodes could be adapted to make that clearer ("translators" instead of "translation", and so on)

Then I have the same sort of problem with edges. What does an edge mean?
* "reports to"? (such as in "project manager" - "advisory committee")
* "is included in"? (such as in "reconciliation services" - "third-party developers")
* "communicates with"? (such as in "advisory committee" - "CS&S")

And then it's not clear to me why certain edges are there and others not. For instance, why is there an edge between "funding partners" and "advisory committee" but not between "funding partners" and "project manager"? Why an edge between "code of conduct committee" and "advisory committee"?

Also I have the feeling that the light blue box is meant to circle where the OpenRefine project starts and ends - but then do we need the central "OpenRefine project" node? Can't that be a label added to the box itself?

Reply:
I removed the diagram and moved this conversation in #325 
I updated the text to include links to the different pages that were referenced in the graphic.